.github/workflows/java_binding_publish.yml

```diff
@@ -0,0 +1,95 @@
+name: Publish Java binding
+
+on:
+  push:
+    branches:
+      - staging
```

Comment:
:carloshuh:

Reply:
what should i replace it with? :carlos:

Reply:
develop

Reply:
But should probably also not run on every push

Reply:
Ok maybe we will get staging :trollface: